### PR TITLE
well-known-hosts: remove RSA keys

### PR DESCRIPTION
**Pull request by zimbatm**
We only need the ed25519 ones

### CODE REVIEW

File: nixos/common/well-known-hosts.nix [hunk 1]
```diff
@@ -2,18 +2,12 @@
...
     github-ed25519.hostNames = [ "github.com" ];
     github-ed25519.publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOMqqnkVzrm0SdG6UOoqKLsabgH5C9okWi0dh2l9GKJl";
 
-    gitlab-rsa.hostNames = [ "gitlab.com" ];
-    gitlab-rsa.publicKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsj2bNKTBSpIYDEGk9KxsGh3mySTRgMtXL583qmBpzeQ+jqCMRgBqB98u3z++J1sKlXHWfM9dyhSevkMwSbhoR8XIq/U0tCNyokEi/ueaBMCvbcTHhO7FcwzY92WK4Yt0aGROY5qX2UKSeOvuP4D6TPqKF1onrSzH9bx9XUf2lEdWT/ia1NEKjunUqu1xOB/StKDHMoX4/OKyIzuS0q/T1zOATthvasJFoPrAjkohTyaDUz2LN5JoH839hViyEG82yB+MjcFV5MU3N1l1QL3cVUCh93xSaua1N85qivl+siMkPGbO5xR/En4iEY6K2XPASUEMaieWVNTRCtJ4S8H+9";
     gitlab-ed25519.hostNames = [ "gitlab.com" ];
     gitlab-ed25519.publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf";
```
Comment (SuperSandro2000) [inline nixos/common/well-known-hosts.nix:9]: If those are renamed to just gitlab.com then the auto completion with bash is nicer and no longer suggests the extra name.

Comment (zimbatm) [reply nixos/common/well-known-hosts.nix]: Like this b9e71438a3500955084ff77e58e7d5ca2c578afb?

Comment (SuperSandro2000) [reply nixos/common/well-known-hosts.nix]: 👍🏼

Comment (zimbatm) [reply nixos/common/well-known-hosts.nix]: thanks

**Comment by zimbatm** [comment]
bors merge

**Pull request merged**